### PR TITLE
Drivers/install: INF Version link correction

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #2038 
i replaced old link to new link
**https://docs.microsoft.com/en-us/windows-hardware/drivers/install/inf-version-section**

### CODE REVIEW

File: windows-driver-docs-pr/install/devpkey-device-classguid.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Requirements
 ## See also
 
 
-[**INF Version Section**](https://docs.microsoft.com/windows-hardware/drivers/install/inf-addservice-directive)
+[**INF Version Section**](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/inf-version-section)
```
Comment (illfated) [inline windows-driver-docs-pr/install/devpkey-device-classguid.md]:
```suggestion
[**INF Version Section**](https://docs.microsoft.com/windows-hardware/drivers/install/inf-version-section)
```
Links should be kept language agnostic, especially if they were so before.

**Comment by illfated** [comment]
Please make the same change (link file name) on line 21:
https://github.com/MicrosoftDocs/windows-driver-docs/pull/2041/files#diff-bcd4274f6733b04c57ac40f93bdf33dfL21
The same link is shown there too.

**Comment by RAJU2529** [comment]
@illfated . Just now i entered below address  
**https://docs.microsoft.com/windows-hardware/drivers/install/inf-version-section**

and above url is  redirected to 

**https://docs.microsoft.com/en-us/windows-hardware/drivers/install/inf-version-section**

so what should i do ? still you prefer without en-us  in url

**Comment by illfated** [comment]
OK, I see. I did not know that you do not know about Microsoft URL language redirection and translated pages.

1. Some of the Microsoft Docs pages are translated to other languages, other pages are not.
2. Even so, if a page is not translated to other languages besides English, you should still get there.
3. Notice that the previous link does not have "/en-us/" as part of its link.
4. It is recommended to follow the existing codestyle and format, unless you can demonstratively and repeatedly show that the link is broken when the language directory part of the link is removed.

The bottom line (TL;DR) is that we should **always** use MS Docs links without a specific language, unless the link breaks.

**Comment by illfated** [comment]
Based on my previous comment, please commit (merge) my PR code suggestion above.

**Comment by illfated** [comment]
Very well, please also edit line 21:

Remarks
-------
The value of DEVPKEY_Device_ClassGuid is set by the INF ClassGUID directive that is supplied by the [**INF Version section**](https://docs.microsoft.com/windows-hardware/drivers/install/inf-addservice-directive) of the INF file that installs a device.

https://github.com/MicrosoftDocs/windows-driver-docs/pull/2041/files#diff-bcd4274f6733b04c57ac40f93bdf33dfR54-R57

`Remarks`
`-------`
`The value of DEVPKEY_Device_ClassGuid is set by the INF ClassGUID directive that is supplied by the [**INF Version section**](https://docs.microsoft.com/windows-hardware/drivers/install/inf-addservice-directive) of the INF file that installs a device.`

**Comment by illfated** [comment]
Also: PR PAGE TITLE suggestion (instead of **replaced wrong link to new link #2041**): 

**Drivers/install: INF Version link correction**
